app/controllers/users_controller.rb

```diff
@@ -20,9 +26,22 @@ def create
     end
   end
 
+  def update
+    @user = User.find_by(params[:id])
+    if @user.update(user_params)
+      redirect_to @user
+    else
+      render 'edit'
```

Comment:
Might be nice to add a flash or something here so that the user has an idea of why their update didn't work.

Also noticing you are using a `.find_by` call here. I think the difference between `.find` and `.find_by` is that `.find_by` doesn't error out if it doesn't find the object so there could be an instance where `@user` is nil on line 31 and calling `.update` would return a NoMethodError anyways. I would like to see consistency between using the two different types of find calls unless there is something I am missing and there is a specific reason you are using `.find_by` in this circumstance.